todo/tests.py

```diff
@@ -76,11 +83,10 @@ def test_update_view_should_update_correct_record(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(self.task0.status, "in_progress")
 
-
-    def test_task_form_view_should_create_correct_record(self):
-        request = self.f.get(reverse('taskform', kwargs={"status": "todo"}))
+    def test_task_form_view_should_have_correct_initial_values(self):
```

Comment:
Is there way to test formview send in request form {"title": ..., "text": ...} and make asertion equals (Task.objects.count(), objects_before_executing_test + 1) ?

Reply:
Is option to get methods of CBV to testing?